tests/test_main.py

```diff
@@ -37,6 +37,23 @@ class Config:
                 include = ["id"]
                 exclude = ["first_name"]
 
+    class AbstractModelSchema(ModelSchema):
+        class Config:
+            abstract = True
```

Comment:
What is expected to happen when `model`, `include`, or `exclude` are set on the config? Need coverage for these cases.

Reply:
I guess it would be best if we started out with some warnings (raise config error) if someone tries to include these on the abstract model to avoid confusion, then if someone later presented a valid use-case to support any of these then maybe could do then if it made sense.